src/problems/objective/mod.rs

```diff
@@ -45,6 +45,7 @@ pub trait Objective: AnyObjective {}
 /// assert!(MultiObjective::try_from(vec![f64::NAN, 0.0]).is_err());
 /// assert!(MultiObjective::try_from(vec![0.0, f64::NEG_INFINITY]).is_err());
 /// ```
+#[allow(missing_docs)]
```

Comment:
My end goal is to add `#[warn(missing_docs)]`, and I added this here because otherwise there's a warning that the enum variants are not documented when the `#[error]` macro already does that. Unfortunately, I forgot that the `error` macro only generates the `Display` implementation, so I'll remove the `allow`.